internal/pkg/skuba/deployments/ssh/testdata/addons/cri/default_flags

```diff
@@ -0,0 +1,7 @@
+## Path           : System/Management
```

Comment:
We should NOT provide this. This should be deleted. We should trust the sysconfig coming from the package instead, and copy it from the package (in `/usr/` iirc), to the `/etc/sysconfig/`.

Reply:
Bear in mind that this is just test data. This is not planned to be installed anywhere.